docs/database-functionalities/import-data.md

```diff
@@ -5,9 +5,9 @@ sidebar_label: Import data
 ---
 
 Memgraph comes with tools for importing data into the database. Data can be
-imported either using the [CSV import tool](#csv-import-tool) or using saved
-[cypher queries](#importing-cypher-queries) with `mg_client` running in non-interactive
-mode.
+imported using the [CSV import tool](#csv-import-tool), using saved [cypher queries](#importing-cypher-queries)
+with `mg_client` running in non-interactive mode, or programatically using the
+`LOAD CSV` Cypher clause.
```

Comment:
Maybe push it to the front of the line because we would like for people to prefer this way (before mg_import_csv)? Or we keep `mg_import_csv` as our main importing tool?